Source: @pullmoll
Upstream: no
Reason: Musl libc already defines PAGE_SIZE

--- libsrc/Dk/tlsf.h	2018-08-16 01:06:21.000000000 +0200
+++ libsrc/Dk/tlsf.h	2020-01-27 15:53:35.498205916 +0100
@@ -159,7 +159,9 @@
 
 #define DEFAULT_AREA_SIZE (1024*10)
 
+#if defined(__GLIBC__)
 #define PAGE_SIZE (getpagesize())
+#endif
 
 #ifndef WIN32
 #ifdef USE_PRINTF
